src/InputField/index.js

```diff
@@ -226,8 +171,8 @@ InputField.propTypes = {
     help: PropTypes.string,
     icon: PropTypes.element,
     trailIcon: PropTypes.element,
-    status: PropTypes.oneOf(['default', 'valid', 'warning', 'error']),
-    size: PropTypes.oneOf(['default', 'dense']),
+    status: iconStatusPropType,
+    size: PropTypes.oneOf([inputSizes.DEFAULT, inputSizes.DENSE]),
     kind: PropTypes.oneOf(['filled', 'outlined']),
```

Comment:
Eventually I would like to redo the `kind` props as well into sharing the scheme better between components.

Reply:
I thought about this as well, but then realized that the components that have a `kind` propType (InputField and SelectField) have different kinds, so I didn't see the necessity to pull this out of the file as that'd be over-engineered code splitting to me.

I think until we see there's shared usage, we don't need to extract them. But I will def. add constants for this, just like for the other values.

Reply:
Okay, nevermind what I said... InputField and SelectField share the same kinds....